data/registers/octospi_v1.yaml

```diff
@@ -0,0 +1,699 @@
+block/OCTOSPI:
+  description: OctoSPI
+  items:
+  - name: CR
+    description: control register
+    byte_offset: 0
+    fieldset: CR
+  - name: DCR1
+    description: device configuration register 1
+    byte_offset: 8
+    fieldset: DCR1
+  - name: DCR2
+    description: device configuration register 2
+    byte_offset: 12
+    fieldset: DCR2
+  - name: DCR3
+    description: device configuration register 3
+    byte_offset: 16
+    fieldset: DCR3
+  - name: SR
+    description: status register
+    byte_offset: 32
+    access: Read
+    fieldset: SR
+  - name: FCR
+    description: flag clear register
+    byte_offset: 36
+    access: Write
+    fieldset: FCR
+  - name: DLR
+    description: data length register
+    byte_offset: 64
+    fieldset: DLR
+  - name: AR
+    description: address register
+    byte_offset: 72
+    fieldset: AR
+  - name: DR
+    description: data register
+    byte_offset: 80
+    fieldset: DR
+  - name: PSMKR
+    description: polling status mask register
+    byte_offset: 128
+    fieldset: PSMKR
+  - name: PSMAR
+    description: polling status match register
+    byte_offset: 136
+    fieldset: PSMAR
+  - name: PIR
+    description: polling interval register
+    byte_offset: 144
+    fieldset: PIR
+  - name: CCR
+    description: communication configuration register
+    byte_offset: 256
+    fieldset: CCR
+  - name: TCR
+    description: timing configuration register
+    byte_offset: 264
+    fieldset: TCR
+  - name: IR
+    description: instruction register
+    byte_offset: 272
+    fieldset: IR
+  - name: ABR
+    description: alternate bytes register
+    byte_offset: 288
+    fieldset: ABR
+  - name: LPTR
+    description: low-power timeout register
+    byte_offset: 304
+    fieldset: LPTR
+  - name: WCCR
+    description: write communication configuration register
+    byte_offset: 384
+    fieldset: WCCR
+  - name: WTCR
+    description: write timing configuration register
+    byte_offset: 392
+    fieldset: WTCR
+  - name: WIR
+    description: write instruction register
+    byte_offset: 400
+    fieldset: WIR
+  - name: WABR
+    description: write alternate bytes register
+    byte_offset: 416
+    fieldset: WABR
+  - name: HLCR
+    description: OCTOSPI HyperBus latency configuration register
+    byte_offset: 512
+    fieldset: HLCR
+  - name: HWCFGR
```

Comment:
these regs at the end of the 1kb peripheral register space are some kind of thing ST started adding around H7 to make peripherals "self describing". They're not specific to this peripheral, and we don't use them anyway. could you remove them?